expr/builtins/url_email.go

```diff
@@ -162,13 +159,11 @@ func domainsEval(ctx expr.EvalContext, args []value.Value) (value.Value, bool) {
 			urlstr = "http://" + urlstr
 		}
 		if urlParsed, err := url.Parse(urlstr); err == nil {
-			parts := strings.Split(urlParsed.Host, ".")
-			if len(parts) > 2 {
-				parts = parts[len(parts)-2:]
-			}
-			if len(parts) > 0 {
-				domains.Append(strings.Join(parts, "."))
+			domain, err := publicsuffix.EffectiveTLDPlusOne(urlParsed.Host)
```

Comment:
tried it without `url.Parse`, but tests fails for `https://www.Google.com/search?q=golang` url. It returns `Google.com/search?q=golang` instead of just `google.com`

Reply:
👍